analyzers/tests/SonarAnalyzer.Test/Helpers/SyntaxHelperTest.cs

```diff
@@ -149,6 +149,25 @@ public void NameIsOrNames_CS(bool expected, params string[] orNames)
             identifier.NameIs("other", orNames).Should().Be(expected);
         }
 
+        [DataTestMethod]
+        [DataRow("Strasse", "Straße")]
+        [DataRow("\u00F6", "\u006F\u0308")] // 00F6 = ö; 006F = o; 0308 = https://www.fileformat.info/info/unicode/char/0308/index.htm
+        public void NameIsCulture_CS(string identifierName, string actual)
```

Comment:
I think the test should mention the expected behavior.
E.g. NameIs_IdentifierComparison_ReturnsFalse

Reply:
From another point of view, is there any reason you did not want to update the `NameIs_CS` that already exists?

Reply:
Both existing tests rely on a fixed SyntaxNode (MemberAccess in `.ToString()` and `IdentifierName("Test")`). I could integrate these tests here, but I think some test isolation is okay in this case. We need at least two UTs anyway (one for each overload for `NameIs`).